packages/map/src/CdcMapComponent.tsx

```diff
@@ -0,0 +1,590 @@
+import { isSolrCsv, isSolrJson } from '@cdc/core/helpers/isSolr'
+import { publish } from '@cdc/core/helpers/events'
+import { generateRuntimeFilters } from './hooks/useGenerateRuntimeFilters'
+import { type MapReducerType, MapState } from './store/map.reducer'
+
+// Map Helpers
+import {
+  addUIDs,
+  closeModal,
+  displayGeoName,
+  formatLegendLocation,
+  getMapContainerClasses,
+  generateRuntimeLegendHash,
+  handleMapTabbing,
+  hashObj,
+  navigationHandler
+} from './helpers'
+
+// Child Components
+import Annotation from './components/Annotation'
+import ConfigContext, { MapDispatchContext } from './context'
+import EditorPanel from './components/EditorPanel'
+import Error from './components/EditorPanel/components/Error'
+import Legend from './components/Legend'
+import Modal from './components/Modal'
+import NavigationMenu from './components/NavigationMenu'
+import UsaMap from './components/UsaMap'
+import WorldMap from './components/WorldMap'
+import GoogleMap from './components/GoogleMap'
+
+// hooks
+import useResizeObserver from './hooks/useResizeObserver'
+import useGenerateRuntimeLegend from './hooks/useGenerateRuntimeLegend'
+import useGenerateRuntimeData from './hooks/useGenerateRuntimeData'
+import { VizFilter } from '@cdc/core/types/VizFilter'
+import { getInitialState, MapReducer, mapReducer } from './store/map.reducer'
+import { RuntimeData } from './types/RuntimeData'
+import EditorContext from '@cdc/editor/src/ConfigContext'
+import MapActions from './store/map.actions'
+import _ from 'lodash'
+
+type CdcMapComponent = {
+  config: MapConfig
+  isDashboard?: boolean
+  isEditor?: boolean
+  link?: string
+  logo?: string
+  navigationHandler: Function
+  setSharedFilter: Function
+  setSharedFilterValue: Function
+}
+
+const CdcMapComponent: React.FC<CdcMapComponent> = ({
+  config: configObj,
+  navigationHandler: customNavigationHandler,
+  isDashboard = false,
+  isEditor = false,
+  logo = '',
+  setSharedFilter,
+  setSharedFilterValue,
+  link,
+  setEditorConfig
+}) => {
+  const initialState = getInitialState(configObj)
+
+  const [mapState, dispatch] = useReducer<MapReducerType<MapState, MapActions>>(mapReducer, initialState as MapState)
+
+  const {
+    loading,
+    displayPanel,
+    runtimeData,
+    runtimeFilters,
+    runtimeLegend,
+    state,
+    modal,
+    accessibleStatus,
+    filteredCountryCode,
+    position,
+    scale,
+    translate,
+    projection,
+    stateToShow,
+    requiredColumns,
+    topoData,
+    coveLoadedHasRan,
+    isDraggingAnnotation
+  } = mapState
+
+  const editorContext = useContext(EditorContext)
+
+  const setState = (newMapState: MapConfig): void => {
+    if (isEditor && !isDashboard) {
+      dispatch({ type: 'SET_STATE', payload: newMapState })
+    } else {
+      dispatch({ type: 'SET_STATE', payload: newMapState })
+      editorContext.setTempConfig(newMapState)
+    }
+  }
+
+  useEffect(() => {
+    setState({ ...state, data: configObj.data })
```

Comment:
This shouldn't be needed in addition to my other changes... I think what was missing was that I didn't refactor the useGenerateRuntimeData function and that may be why the map data wasn't updating.